examples/pennylane_run_quantum_natural_gradient.py

```diff
@@ -0,0 +1,356 @@
+the set of quantum parameters that *minimize* a particular measurement statistic
+of the quantum device. Examples of such algorithms include the :ref:`variational quantum
+eigensolver (VQE) <vqe>`, the `quantum approximant optimization algorithm (QAOA) <https://arxiv.org/abs/1411.4028>`__,
+and :ref:`quantum neural networks (QNN) <quantum_neural_net>`.
+
+Due to the inherent noise of near-term quantum hardware, most recent implementations
+of variational quantum algorithms have used gradient-free classical optimization
+methods, such as Nelder-Mead. However, the parameter-shift rule for
+analytic quantum gradients (as implemented in PennyLane) has allowed for
+stochastic gradient descent of variational quantum algorithms on quantum
+hardware. Once caveat though that has been surfaced with gradient descent
+is the issue of learning rate or step-size --- how do we choose the optimal
+step size for our variational quantum algorithms, to ensure successful and
+efficient optimization?
+
+The natural gradient
+^^^^^^^^^^^^^^^^^^^^
+
+In standard gradient descent, each optimization step is given by
+
+.. math:: \theta_{t+1} = \theta_t -\eta \nabla \mathcal{L}(\theta),
+
+where :math:`\mathcal{L}(\theta)` is the loss as a function of
+the parameters :math:`\theta`, and :math:`\eta` is the learning-rate
+or step size. In essence, each optimization step calculates a vector of
+steepest descent direction around the local value of :math:`\theta_t`
+in the parameter space, and updates :math:`\theta_t\rightarrow \theta_{t+1}`
+in by this vector in euclidean space.
+
+The problem with the above approach is that each optimization step
+is highly dependent on the euclidean geometry of the parameter space.
+The parametrization chosen is not necessarily unique; the scale of the
+parameters, or even the parametrization used, could easily be modified.
+If we instead consider the optimization problem as a distribution of possible
+output values given an input, a better approach is to instead perform the
+gradient descent in the *distribution space*, which is invariant with respect
+to the parametrization.
+
+In classical neural networks, the above process is known as
+*natural gradient descent*, and was first introduced by
+`Shun-Ichi Amari (1998) <https://www.mitpressjournals.org/doi/abs/10.1162/089976698300017746>`__.
+The standard gradient descent is modified as follows:
+
+.. math:: \theta_{t+1} = \theta_t - \eta F^{-1}\nabla \mathcal{L}(\theta),
+
+where :math:`F` is the `Fisher information matrix <https://en.wikipedia.org/wiki/Fisher_information#Matrix_form>`__.
+The Fisher information matrix acts as a metric tensor, transforming the
+steepest descent in the euclidean parameter space to the steepest descent in the
+distribution space.
+
+The quantum analogue
+^^^^^^^^^^^^^^^^^^^^
+
+In a similar vein, it has been shown that the standard euclidean geometry
+is sub-optimal for optimization of quantum variational algorithms.

```

Comment:
why?

Reply:
Or where? ;P

Reply:
Added in a citation to https://arxiv.org/abs/1901.05374 :slightly_smiling_face: